flink-metrics/flink-metrics-datadog/src/test/java/org/apache/flink/metrics/datadog/DatadogHttpClientTest.java

```diff
@@ -53,14 +59,37 @@ public void mockSystemMillis() {
 		PowerMockito.when(DMetric.getUnixEpochTimestamp()).thenReturn(MOCKED_SYSTEM_MILLIS);
 	}
 
+	@Before
+	public void suppressValidateApiKey(){
+		PowerMockito.suppress(MemberMatcher.method(DatadogHttpClient.class, "validateApiKey"));
```

Comment:
I'd rather catch the exception and continue with the test.

Reply:
I'm not sure how catching the exception and continuing should work because validateApiKey is called in the constructor, so we couldn't then call getProxy to validate that value. If just ensuring that getProxy doesn't error out is enough, then I can wrap testGetProxyWithNullProxyHost and testGetProxy in try-catches.

Reply:
ah of course, that makes sense...let's leave it as is.